activities/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.conf.urls import url, include
 
 from activities import views
+from django.views.decorators.csrf import csrf_exempt
```

Comment:
unused import

Reply:
Code outdated. It has already been removed